pywikigraph/wikigraph.py

```diff
@@ -44,6 +48,13 @@ def __init__(self, data_root: Optional[str] = None, optimize_memory: bool = True
     def adj_matrix_directed_csr(self) -> Optional[csr_matrix]:
         """Directed adjacency matrix of Wikipedia topics in sparse CSR format."""
         if self._adj_matrix_directed_csr is None:
+            if not os.path.exists(
+                os.path.join(self.data_root, "wikigraph_directed_csr.npz")
+            ):
+                urllib.request.urlretrieve(
```

Comment:
perhaps we should add a print or log message since the download may be time-consuming... (same with the one below)